src/deploy/missions/contracts/z-token/z-token.ts

```diff
@@ -86,35 +86,36 @@ IZNSContracts
   }
 
   getArtifact () {
-    return meowArtifact;
+    return zArtifact;
   }
 
+  // it will choose the governon as `admin` argument
+  // and deployAdmin as `minter` and first passed admin as `mintBeneficiary`.
   async deployArgs () : Promise<TDeployArgs> {
-    return [meowTokenName, meowTokenSymbol];
+    const {
+      initialAdminDelay,
+      initialSupplyBase,
+      inflationRates,
+      finalInflationRate,
+    } = this.config.zTokenConfig as unknown as IZTokenConfig;
```

Comment:
We don't need to use `as unknown` here, this object complies with the definition of `IZtokenConfig` fine